server/config.ts

```diff
@@ -32,6 +32,7 @@ export interface ApiConfig {
 export default {
   production,
   https: production,
+  enableApplicationInsights: production,
```

Comment:
Is it worth explaining why `production-type environments` means dev, preprod and prod in the commit message? Maybe the first sentence is enough